metadata.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: blueprints.cloud.google.com/v1alpha1
```

Comment:
No headers are present in the file. This will fail the lint test. I see you have already set ENABLE_BPMEDATA to `1`. Try running `make generate_docs` again.

Reply:
Ran `make generate_docs` again. PTAL